openapi/generated.json

```diff
@@ -4510,21 +4613,5 @@
     {
       "name": "Payment Methods"
     }
-  ],
-  "securitySchemes": {
-    "ApiSecretKey": {
-      "description": "Unless explicitly stated, all endpoints require authentication using your secret key.  You may generate your API keys from the Juspay Dashboard.\n#### Format\n- Sandbox `sk_xxxxxxxxxxxxxxxxxxxxxxxxxx`\n- Production `pk_xxxxxxxxxxxxxxxxxxxxxxxxxx`\n",
-      "name": "api-key",
-      "type": "apiKey",
-      "in": "header",
-      "x-cko-type": "api-key"
-    },
-    "AdminSecretKey": {
-      "description": "Admin Key for managing Merchant Accounts and Payment Connectors In the OSS version - Developers can directly update the AdminSecretKey in the DB and start using the admin APIs to configure a new accounts and connectors In the Hosted version - The key can be obtained from the Dashboard or the Support team\n",
-      "name": "admin-api-key",
-      "type": "apiKey",
-      "in": "header",
-      "x-cko-type": "admin-api-key"
-    }
-  }
```

Comment:
This whole block and the usages of these security schemes was lost in this commit. Is this intentional?

Reply:
Had manually added this block to the generated.json in the previous pull. Will need to add this later in the openapi.rs and other routes files so that it gets generated automatically. Will raise a separate PR for this next week after discussion